test/WebSites/UserClassLibrary/UserClassLibrary.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard1.3</TargetFrameworks>
+    <AssemblyName>UserClassLibrary</AssemblyName>
```

Comment:
Nit: both AssemblyName and PackageId aren't required here. These are the default values.